tests/accessor_legacy/accessor_api_common_buffer_local.h

```diff
@@ -50,7 +50,7 @@ struct explicit_pointer<T, sycl::target::local> {
 */
 template <typename T>
 struct explicit_pointer<T, sycl::target::device> {
```

Comment:
Don't we need to modify specializations for `sycl::target::local` and `sycl::target::constant_buffer` too?

Reply:
No, these targets uses correct types due to spec.